packages/gestalt/src/__snapshots__/Avatar.test.js.snap

```diff
@@ -110,7 +110,7 @@ exports[`Avatar renders the Pinterest P on verified 1`] = `
   }
 >
   <div
-    className="Mask circle"
+    className="Mask circle hintBrowser"
```

Comment:
hmm not crazy that we're introducing this classname in all the tests, maybe there's another way?  Ultimate I need to turn off `will-change: transform` for Safari full screen, but there's not an easy way to use css selectors for Safari: https://stackoverflow.com/questions/16348489/is-there-a-css-hack-for-safari-only-not-chrome.  And the element that is full screen is a child of `Mask`

Reply:
It's totally fine that it's added to all the snapshots.